lms/djangoapps/commerce/api/v1/tests/test_views.py

```diff
@@ -174,8 +174,9 @@ def test_update(self):
         self.assertIsNone(VerificationDeadline.deadline_for_course(self.course.id))
 
         # Generate the expected data
-        verification_deadline = datetime(year=2020, month=12, day=31, tzinfo=pytz.utc)
-        expiration_datetime = datetime.now(pytz.utc)
+        now = datetime.now(pytz.utc)
+        verification_deadline = now + timedelta(days=1)
+        expiration_datetime = now
```

Comment:
```suggestion
        expiration_datetime = now - timedelta(days=10)
```

Reply:
Thank you @OmarEdraak . Your suggestions are better. But the fix is copied from upstream https://github.com/edx/edx-platform/blob/e17e691d54c44590fdf24a684ce0c996f8adc668/lms/djangoapps/commerce/api/v1/tests/test_views.py#L178 . It doesn't worth upgrade conflicts, what do you think?

Reply:
yes. let's keep it as-is. Thanks for the reminder Shadi.